### DISCUSSION TITLE
Docs: "next/experimental/testmode/playwright"' has no exported member 'rest'

### DISCUSSION THREAD

**Discussion by FransJoakim** [vercel/next.js#71330] [Help]
### What is the documentation issue?

I'm writing e2e tests and need to intercept api calls made to fetch data server-side. I'm trying out the Experimental test mode for Playwright, however when implementing [this example shown in the docs](https://github.com/vercel/next.js/blob/canary/packages/next/src/experimental/testmode/playwright/README.md#or-use-the-nextexperimentaltestmodeplaywrightmsw) I get this error: "Module '"next/experimental/testmode/playwright"' has no exported member 'rest'." When running the code rest also evaluates to undefined. 

### Is there any context that might help us understand?

I'm using 
* "next": "14.2.12",
* "msw": "^2.4.11",
* "@playwright/test": "^1.41.2",

### Does the docs page already exist? Please link to it.

https://github.com/vercel/next.js/blob/canary/packages/next/src/experimental/testmode/playwright/README.md#or-use-the-nextexperimentaltestmodeplaywrightmsw

**Comment by ariawisp** [comment]
Next.js 14 depends on msw 1.3.0, while Next.js 15 depends on msw 2.3.0. `rest` was changed to `http` in msw v2. The documentation page was written for Next.js 14 + msw 1.3.0.
